infra/util/test/TestAllocatorFixedSpace.cpp

```diff
@@ -13,6 +13,7 @@ class FixedSpaceAllocatorTest
 
 TEST_F(FixedSpaceAllocatorTest, allocate_one_object)
 {
+    EXPECT_TRUE(false);
```

Comment:
Is this a test to see if we pay attention 😅.

Reply:
:sweat_smile: No I wanted to see the workflow storing the results on a failed run